cfme/services/requests.py

```diff
@@ -247,7 +247,7 @@ def in_requests(self):
 
 
 class RequestsView(RequestBasicView):
-    table = Table(locator='//*[@id="list_grid"]/table')
+    table = Table(locator='//div[@id="gtl_div"]//table')
```

Comment:
**REQUIRED**
Fixed here https://github.com/ManageIQ/integration_tests/pull/5833/files#diff-cbcd422f5444a1c4d39c88ca3f3aa7bfR251

Reply:
but this locator covers both situations  - no version picking and it's better to have it separately - it's a fix not related to other code @quarckster

Reply:
Cool, the locator indeed is more versatile. Thanks.